.github/workflows/publish.yml

```diff
@@ -37,127 +38,157 @@ jobs:
               exit 1
             fi
           else
-            # Need to keep separate DATE var b/c $(date -d) can't parse y.m.d
-            DATE=$(date)
-            EXT_VERSION=$(date -d "$DATE" +%y.%-m.%-d);
+            EXT_VERSION=$(date +%y.%-m.%-d);
 
             while git rev-parse "refs/tags/$EXT_VERSION" >/dev/null 2>&1; do
               echo "Tag $EXT_VERSION already exists. Incrementing day."
               DATE=$(date -d "$DATE + 1 day")
               EXT_VERSION=$(date -d "$DATE" +%y.%-m.%-d);
             done;
-
-            echo "Setting extension version to $EXT_VERSION"
-            echo "ext-version=$EXT_VERSION" >> $GITHUB_OUTPUT
           fi
-
-  CD:
-    runs-on: ubuntu-latest
-    needs:
-      - setup
-    strategy:
-      matrix:
-        node-version: [18.x]
-    env:
-      EXT_VERSION: ${{ needs.setup.outputs.ext-version }}
-    steps:
-      - name: Authorized by
-        run: echo "${{ github.event.sender.login }}"
-
-      - uses: actions/checkout@v3
-        with:
-          fetch-depth: '0'
-
-      - name: Set up Node
-        uses: actions/setup-node@v3
-        with:
-          node-version: ${{ matrix.node-version }}
-          cache: 'npm'
-
-      - name: Update manifest
-        run: |
-          echo "New Version: ${{ env.EXT_VERSION }}"
-          sed -i'' -e 's|"version":.*|"version": "${{ env.EXT_VERSION }}",|' src/manifest.json
-          cat src/manifest.json
-
-      - name: Install dependencies
-        run: npm ci
-
-      - name: Build
-        run: npm run build
-
-      - uses: actions/upload-artifact@v3
-        with:
-          name: release-${{ env.EXT_VERSION }}
-          path: ./dist/*
-
-      - name: Zip it! zip it good!
-        id: create-zip
-        run: |
-          mkdir build
-          zip -r ./build/release-${{ env.EXT_VERSION }}.zip ./dist/*
-
-      - uses: ncipollo/release-action@v1
-        with:
-          name: ${{ env.EXT_VERSION }}
-          tag: ${{ env.EXT_VERSION }}
-          artifacts: ./build/release-${{ env.EXT_VERSION }}.zip
-          generateReleaseNotes: true
-          token: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Tag with new version
-        id: create-tag
-        run: |
-          git tag ${{ env.EXT_VERSION }}
-          git push --tags
-
-  chrome-submit:
-    name: Submit to Chrome
+          echo "Setting extension version to $EXT_VERSION"
+          echo "ext-version=$EXT_VERSION" >> $GITHUB_OUTPUT
+          # Remove dots and pad to two digits and remove spaces
+          EXT_VERSION_NUM=$(echo $EXT_VERSION | sed 's/\./ /g' | xargs -n1 | awk '{printf "%02d", $1;}' | xargs)
```

Comment:
It may be easier to understand to just use `date` again and supply a new format string.

With `+%y.%-m.%-d` (from above), `-` (hyphen) says do not pad the field

So you can just do `date +%y%m%d ` and get `230916`

Reply:
this is not always date. this is sometimes custom date, sometimes date + 1

Reply:
Oh true. Good catch.